code/game/machinery/requests_console.dm

```diff
@@ -497,8 +590,24 @@ var/list/obj/machinery/requests_console/requests_consoles = list()
 			var/obj/item/weapon/stamp/T = O
 			msgStamped = text("<font color='blue'><b>Stamped with the [T.name]</b></font>")
 			updateUsrDialog()
-	return
+	if (istype(O, /obj/item/telephone) && landline)
+		landline.attackby(O, user)
+
+/obj/machinery/requests_console/verb/pick_up_phone()
+	set category = "Object"
+	set name = "Pick up telephone"
+	set src in view(1)
+	if(!landline)
+		to_chat(usr, "<span class='notice'>\the [src] model does not come with a telephone!</span>")
+		return
+	landline.pick_up_phone(usr)
 
+/obj/machinery/requests_console/CtrlClick(mob/user)
```

Comment:
This really should be an alt click rather than ctrl click; also should check if the user is incapacitated

Reply:
incapacitated check is in /obj/landline/proc/pick_up_phone

Reply:
Why alt-click over a ctrl-click?

Reply:
Ctrl-click isn't used on machines that often and is pretty much exclusively used for dragging. Alt-click has interactions with other machines (unlocking APCs and air alarms, radial menus on several machines, etc), and RC's don't currently have any alt-click functionality it would interfere with.